packages/venia-ui/lib/components/AccountMenu/accountMenuItems.module.css

```diff
@@ -1,26 +1,26 @@
 .root {
-    display: grid;
-    grid-auto-flow: row;
-    justify-content: stretch;
-    row-gap: 1.5rem;
-    padding: 1.5rem;
+    composes: gap-y-sm from global;
+    composes: grid from global;
+    composes: grid-flow-row from global;
+    composes: justify-stretch from global;
+    composes: p-sm from global;
 }
 
 .link {
     composes: root from '../LinkButton/linkButton.module.css';
```

Comment:
Agree. We should get rid of all cross-component dependencies, since Tailwind handles deduplication for us now automatically. If we really need some shared CSS, we can write a Tailwind plugin.